params.go

```diff
@@ -41,8 +41,8 @@ type ContactsParams struct {
 	Query string `json:"query,omitempty"`
 }
 
-// The OAuth Tokens Struct returned from OAuth Authentication
-type oauthTokens struct {
+// OauthTokens the OAuth Tokens Struct returned from OAuth Authentication
```

Comment:
Nit: In my opinion, this change does not add value to the comment. Since the `OauthTokens` struct is on the next line, adding its name to this comment does not seem to add information to this comment. I would remove this, but will approve anyways as it's not hurting anything and is subjective.